Trading/Exchange/bittrex/bittrex_exchange.py

```diff
@@ -33,7 +33,7 @@ def get_name(cls):
     def is_supporting_exchange(cls, exchange_candidate_name) -> bool:
         return cls.get_name() == exchange_candidate_name
 
-    async def get_order_book(self, symbol, limit=5, **kwargs):
+    async def get_order_book(self, symbol, limit=DEFAULT_ORDER_BOOK_LIMIT, **kwargs):
```

Comment:
change from 5 to 25 ?

Reply:
As 5 is not supported, we should directly set the default limit to 25